subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -130,6 +132,17 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	if (num_bis != 1) {
 		return BT_HCI_ERR_MEM_CAPACITY_EXCEEDED;
 	}
+
+	/* Add ACAD to AUX_SYNC_IND */
+	err = ull_adv_sync_acad_enable(lll_adv_sync,
+				       (sizeof(struct pdu_big_info) + 2),
+				       (void **)&acad);
+	if (err) {
+		return err;
+	}
+	acad[0] = sizeof(struct pdu_big_info) + 1;
+	acad[1] = BT_DATA_BIG_INFO;
```

Comment:
Would it make sense to also add the AD type to the list of parameters for `ull_adv_sync_acad_enable`? It seem that if we do that, then `ull_adv_sync_acad_enable` could be responsible for settings the AD type as well as the length field.

Reply:
ACAD can have mutiple advertising data types in a length encoded format. Hence, it is best left to the caller of the function to populate the ACAD with multiple data types.